lib/client/api.go

```diff
@@ -4308,3 +4319,16 @@ func findActiveDatabases(key *Key) ([]tlsca.RouteToDatabase, error) {
 	}
 	return databases, nil
 }
+
+// getDesktopEventWebURL returns the web UI URL users can access to
+// watch a desktop session recording in the browser
+func getDesktopEventWebURL(proxyHost string, cluster string, sid *session.ID, events []events.EventFields) string {
+	if events == nil || len(events) < 1 {
+		return ""
+	}
```

Comment:
```suggestion
func getDesktopEventWebURL(proxyHost string, cluster string, sid *session.ID, events []events.EventFields) string {
	if len(events) < 1 {
		return ""
	}
```
nit: you can just use the `len` nil slices in go